src/main/resources/static/css/style.css

```diff
@@ -261,3 +261,61 @@ a:hover{
 .alert-light {
     background-color: #f3f3f3;
 }
+
+/** Login page */
+.login-page-body {
+    background-image: url('/img/login-background-small.jpg');
+    background-repeat: no-repeat;
+}
+/* Provide different image sizes for different screens to save bandwidth*/
+@media (min-width: 576px) {
+    .login-page-body {
+        background-image: url('/img/login-background-large.jpg');
+    }
+}
+@media (min-width: 768px) {
+    .login-page-body {
+        background-image: url('/img/login-background-large.jpg');
+    }
+}
+
+@media (min-width: 1200px) {
+    .login-page-body {
+        background-image: url('/img/login-background-large.jpg');
+    }
+}
+@media (min-width: 992px) {
```

Comment:
Tady je spatne poradi. Pokud tam chces nechat i tento hranicni bod, pak musi byt pred `@media (min-width: 1200px) {`

Reply:
Aha, prehozeno. No nechavat to tam nemusim, zalezi jestli se s tema obrazkama jeste neco udela.